aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -186,11 +186,11 @@ Tensor expand_batching_rule(const Tensor& self, IntArrayRef size, bool implicit)
 }
 
 Tensor expand_symint_batching_rule(const Tensor& self, SymIntArrayRef psize, bool implicit) {
-  return expand_batching_rule(self, asIntArrayRefSlow(psize), implicit);
```

Comment:
calling the batching rules directly here instead of re-invoking the dispatcher is probably fine TBH.